cli/src/commands/logs.ts

```diff
@@ -0,0 +1,39 @@
+import { Command, flags } from '@oclif/command';
+
+export default class Logs extends Command {
+  public static description = 'Show logs';
+
+  public static examples = [
+    '$ shift-cli logs',
+    '$ shift-cli logs --follow',
+    '$ shift-cli logs --since 2018-03-09T22:12:21.861Z',
+    '$ shift-cli logs --since 3d',
+    '$ shift-cli logs --since 13hours',
+    '$ shift-cli logs --since 9s*',
+    '$ shift-cli logs --since 2m --follow',
+  ];
+
+  public static flags = {
+    help: flags.help({char: 'h'}),
+    limit: flags.integer({
+      char: 'l',
+      description: 'Limit number of entries shown (cannot exceed 1000).',
```

Comment:
Where do we enforce the maximum?

Reply:
in the command code I guess.
but this will be in the future PR with the logs implementation

Reply:
There's no way to provide validation of this value with the integer() method?

Reply:
add custom flags for validation and parsing